docs/virtual_machines/interfaces_and_networks.md

```diff
@@ -518,6 +518,29 @@ fields.
 More information about SLIRP mode can be found in [QEMU
 Wiki](https://wiki.qemu.org/Documentation/Networking#User_Networking_.28SLIRP.29).
 
+#### Slirp network binding plugin
+Kubevirt delegate Slirp network configuration to a hook sidecar using Slirp network binding plugin. 
+The sidecar container is added to the VM virt-launcher pod using the registered Slirp network binding plugin image in Kubevirt config.
+
+To register Slirp network binding plugin image, specify the image in Kubevirt config as follows:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: KubeVirt
+metadata:
+  name: kubevirt
+  namespace: kubevirt
+spec:
+  configuration:
+    network:
+      binding:
+        slirp:
+          sidecarImage: "kubevirt/network-slirp-plugin"
+```
+
+In case no image is registered, Kubevirt will use the following default image: `quay.io/kubevirt/network-slirp-binding:20230830_638c60fc8`.
+
+> **Note:** In the next release (v1.2.0) no default image will be set by Kubevirt, registering an image will be mandatory.
+
```

Comment:
Consider adding a note about the need to have access to this image.

Reply:
Done